botocore/args.py

```diff
@@ -259,6 +266,27 @@ def compute_client_args(
             ),
         }
 
+    def _compute_configured_endpoint_url(self, client_config, endpoint_url):
+        if self._ignore_configured_endpoint_urls(client_config):
+            logger.debug("Ignoring configured endpoint URLs.")
+            return endpoint_url
+
+        if endpoint_url is not None:
+            return endpoint_url
+
+        return self._config_store.get_config_variable('endpoint_url')
+
+    def _ignore_configured_endpoint_urls(self, client_config):
+        if (
+            client_config
+            and client_config.ignore_configured_endpoint_urls is not None
```

Comment:
We should make sure there are functional tests that ensure we obey the `ignore_configured_endpoint_urls` from the `client_config`. I forgot that this was an available interface when going over the the functional test PR

Reply:
https://github.com/boto/botocore/pull/2974

Reply:
Merged #2974.